inst/tests/tests.Rraw

```diff
@@ -1489,6 +1489,16 @@ test(498.3, setcolorder(DT, 1:4), error = "specify non existing column*.*4")
 # Test where neworder=NULL, thus ordered by key and index columns
 DT = data.table(a = 1:3, b = 2:4, c = 3:5, d = 4:6, key="b")
 test(498.4, names(setcolorder(DT)), c("b", "a", "c", "d"))
+# Test setcolorder with parameter of anchor
+DT = data.table(a=1,b=2,c=3)
+test(498.5, setcolorder(copy(DT), "c", c(after="a")), data.table(a=1,c=3,b=2))
```

Comment:
The builds are erroring based on the test numbering. I believe it will work if you use 498.51, 498.52, 498.53, etc. Alternatively, you can add your tests at the very end with new numbers. 

While code coverage can wait as the API is being ironed out, you may want to add a test for a non-integer or character anchor as well as specify the anchor as also being the new order. 

```r
setcolorder(copy(DT), 1L, c(before = 1L))
setcolorder(copy(DT), 1L, c(before = list(1L)))
```